src/app/features/genes/components/gene-evidence-rna/gene-evidence-rna.component.ts

```diff
@@ -71,13 +77,20 @@ export class GeneEvidenceRnaComponent implements AfterViewInit {
     this.initConsistencyOfChange();
   }
 
-  ngAfterViewInit() {
-    const hash = window.location.hash.substr(1);
-    if (hash) {
-      const target = document.getElementById(hash);
-      if (target) {
-        // TODO determine if there are async calls altering the offset height
-        window.scrollTo(0, this.helperService.getOffset(target).top - 150);
+  ngAfterViewChecked() {
+    this.scrollToAnchorLink();
+  }
```

Comment:
Moved into `ngAfterViewChecked`, since the boxplot hasn't fully loaded when `ngAfterViewInit` is called. Since `ngAfterViewChecked` can be called multiple times, track whether we've already scrolled with `hasScrolled`, so that the page isn't scrolled to the anchor link multiple times.